content/src/view.js

```diff
@@ -1350,7 +1351,47 @@ $(window).on('resize.listing', function() {
   }
 });
 
-function updatePaneLinks(pane) {
+
+var searchCache = "file-search";
+var prevSearch;
+function updateSearchResults(search,pane) {
+  search = search.toLowerCase();
+  if (search == prevSearch){
+    return;
+  }
+    
+  if (!state.pane[pane].allLinks){
+    state.pane[pane].allLinks = state.pane[pane].links;
+  }
+    
+  var results;
+  if (search) {
+    var savedCache = cache.get(searchCache,search);
```

Comment:
In this case, I don't see that we need a cache.  We are better off just filtering each time when we need to.

Even for server-based searches, I'd argue avoid a cache until we find that we really need one.  Caches (and cache inconsistency) is one of the top sources of bugs.